scrapper.py

```diff
@@ -25,39 +39,146 @@ class Crawler:
     """
     Crawler implementation
     """
-    def __init__(self, seed_urls, max_articles: int):
-        pass
+
+    def __init__(self, seed_urls, total_max_articles: int):
+        self.seed_urls = seed_urls
+        self.total_max_articles = total_max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        article_summaries_bs = article_bs.find_all("div", class_="obj_article_summary")
+        urls_bs = [article_summary_bs.find('div', class_='title').find(
+            'a')['href'] for article_summary_bs in article_summaries_bs]
+        self.urls.extend(urls_bs)
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+
+        for url in self.seed_urls:
+            response = requests.get(url)
+            sleep_period = random.randint(1, 3)
+            sleep(sleep_period)
+
+            if not response.ok:
+                continue
+
+            soup = BeautifulSoup(response.text, 'lxml')
```

Comment:
I did not see lxml in your project dependencies

Reply:
Added